src/actions/index.js

```diff
@@ -0,0 +1,12 @@
+const createBook = book => ({
+  type: 'CREATE_BOOK',
```

Comment:
**OPTIONAL**: You are using constants on your reducers for the actions, you can use those here instead of typing the action. Usually, you would have those constants here as those are the actions, and use them here on your actions creators and import them on any reducers using those actions to avoid errors (typos).

Reply:
Done :D